gluster-exporter/metric_volume.go

```diff
@@ -173,8 +175,132 @@ var (
 		LongHelp:  "",
 		Labels:    volumeProfileFopInfoLabels,
 	})
+
+	glusterVolumeProfileFopTotalHitsAggregatedOps = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "volume_profile_fop_total_hits_on_aggregated_fops",
+		Help: "Cumulative total hits on aggregated FOPs" +
+			" like READ_WRIET_OPS, LOCK_OPS, INODE_OPS etc",
+		LongHelp: "",
+		Labels:   volumeProfileFopInfoLabels,
+	})
+
+	glusterVolumeProfileFopTotalHitsAggregatedOpsInt = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "volume_profile_fop_total_hits_on_aggregated_fops_interval",
+		Help: "Interval based total hits on aggregated FOPs" +
+			" like READ_WRIET_OPS, LOCK_OPS, INODE_OPS etc",
+		LongHelp: "",
+		Labels:   volumeProfileFopInfoLabels,
+	})
 )
 
+// OpType represents aggregated operations like
+// READ_WRITE_OPS, INODE_OPS, ENTRY_OPS, LOCK_OPS etc...
+type OpType struct {
```

Comment:
Do you need these functions to be public at all? Are they used anywhere else?

Reply:
Don't require them to be public. Only reasoning was for a future case when we move them to a separate package, we have to just copy paste the code as is to the new place.

As discussed, will lower case the names.